procfile/README.md

```diff
@@ -42,3 +42,13 @@ Specifies the command that should be run when executing this process
 ```yaml
 command: ./bin/web
 ```
+
+**Expose**
+
+Specifies the exposure settings for the process. This allows you to expose a tcp/http/https/ssl service on any process in the application.
+
+```yaml
+expose:
+  type: tcp // Possible options are tcp/http/https/ssl
```

Comment:
i like having the ability for each port mapping to define its protocol.

you could also support inference for the http/https use case just because it is pretty common.

ie:

```
web:
  visibility: public
  ports:
    - 80:8080
    - 443:8080
    - 4321:8081
      protocol: tcp
```